AprilSylph/XKit-Rewritten
# Quick Tags: Implement New XKit migration

### Description


Along with https://github.com/new-xkit/XKit/pull/2135, this is one way of implementing a button that migrates data from New XKit to XKit Rewritten when pressed. It communicates via custom events from one extension's content script to the other. 

### Testing steps


- Load the corresponding New XKit PR, e.g. via the XKit editor.
- Create at least one unique New XKit tag bundle.
- Enable Quick Tags in XKit Rewritten.
- Press the button in the New XKit Quick Tags settings pane. Confirm that the tag bundle appears in XKit Rewritten.
